src/app/file/file.js

```diff
@@ -72,6 +72,9 @@ angular.module('app.file', [
   var self = this;
   $scope.$parent.selected = {id: file.id};
   self.file = file;
+  // set display options
```

Comment:
Can you add a more accurate comment here. "Specifying which parameters of the file to display" would be more accurate.